upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -900,6 +900,24 @@ print_subsection
 verify_vlan
 }
 
+function verify_pid_limit(){
+    print info "Verify PID Limit"
+    rm -f ${TEMP_MMHEALTH_FILE} >> /dev/null
+    while read -r proc; do echo $proc >> ${TEMP_MMHEALTH_FILE}; done <<< "$(oc get nodes --no-headers |awk '{print $1}')"
+    while IFS= read -r line
+    do
+        pidvalue=$(oc debug nodes/$line -- chroot /host grep podPidsLimit /etc/kubernetes/kubelet.conf)
+        count=$(echo $pidvalue | grep -c "12228")
```

Comment:
we should not be checking for exact 12228, we should get current value and fail if value is lesser than 12228, higher value is also success and good and we should not fail that pidslimit is not right

Reply:
for any command that you add try to send it's output to dev/null so we do not have extra text on console when script is being run

Reply:
modified the check to fail if value is lower than 12228 and also added changes to send the output to /dev/null